### PR TITLE
Dev v2.11.2  For August 2016 Release

### PR DESCRIPTION
**Pull request by ronaldddilley**

### CODE REVIEW

File: processing/test-orders/run-ondemand-request.py [hunk 1]
```diff
@@ -290,6 +290,52 @@ def export_environment_variables(cfg):
...
+    """Build the full path to the config file
+
+    Args:
+        filename <str>: The name of the file to append to the full path
+
+    Raises:
+        Exception(<str>)
+    """
+
+    # Use the users home directory as the base source directory for
```
Comment (raydittmeier) [inline processing/test-orders/run-ondemand-request.py]: Nitpicking: "user's" instead of "users"

Comment (ronaldddilley) [inline processing/test-orders/run-ondemand-request.py]: Fixed

**Comment by gschmidt-usgs** [comment]
Code review complete.  No comments.

File: processing/cli.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
+#! /usr/bin/env python
+
```
Comment (caustin-usgs) [inline processing/cli.py]: doc string here would be nice, describing the purpose of this module

Comment (ronaldddilley) [inline processing/cli.py]: I intend to have one for it, but at this point it is only a prototype command line interface for use with Docker.
It maybe replaces or uses ondemand_mapper.py when we go to mesos or whatever.....

**Comment by gschmidt-usgs** [comment]
Reviewed the latest changes and no comments.

**Pull request merged**